src/cryptoadvance/specter/config.py

```diff
@@ -112,6 +119,9 @@ class DevelopmentConfig(BaseConfig):
         os.getenv("SPECTER_DATA_FOLDER", "~/.specter_dev")
     )
 
+    # Env vars take priority over config settings so ensure that this is set
```

Comment:
Does this really work? I can loosely remember that i struggeled to define ENV-vars like this because it's already too late for that. But i might be wrong.

Reply:
In my testing it didn't work if I just declared the var:
```
FLASK_ENV = "development"
```
But this `os.environ` approach has seemed to work for me.